contracts/EventStore.sol

```diff
@@ -32,35 +41,6 @@ contract EventStore is Killable {
   uint public solidityEventCount;
   mapping (uint => SolidityEvent) solidityEvents;
 
-  uint public eventCount;
-  mapping (uint => TransmuteEvent) events;
-
-  event SOLIDITY_EVENT_PROPERTY(
-    uint EventIndex,
-    uint EventPropertyIndex,
-    string Name,
-    string Type,
-    address AddressValue,
-    uint UIntValue,
-    string StringValue
-  );
-
-  event SOLIDITY_EVENT(
-    uint Id,
-    string Type,
-    uint Created,
-    uint PropertyCount,
-    string IntegrityHash
-  );
-
-  event NEW_EVENT(
-    uint Id,
-    string Type,
-    uint Created,
-    address AddressValue,
-    uint UIntValue,
-    string StringValue
-  );
 
   function () payable {}
   function EventStore() payable {}
```

Comment:
@OR13 I'm going to be compulsive on this because open source. I think we should always have line carriages between methods. Hopefully, we'll have docs in the code anyways that prevents two signatures from being next to one another, but just for the future.